sunkit_pyvista/tests/test_pyvista.py

```diff
@@ -101,6 +101,10 @@ def test_plot_coordinates(plotter):
     expected_center = [-0.5000000149011612, -0.5, 0.7071067690849304]
     assert np.allclose(plotter.plotter.mesh.center, expected_center)
 
+    pixel_pos = np.argwhere(aia171_test_map.data == aia171_test_map.data.max()) * u.pixel
+    hpc_max = aia171_test_map.pixel_to_world(pixel_pos[:, 1], pixel_pos[:, 0])
+    plotter.plot_coordinates(hpc_max, color='blue')
```

Comment:
Do we not want to check this worked somehow?

Reply:
yikes, I forgot to push that commit, my apologies